pymeasure/instruments/instrument.py

```diff
@@ -219,3 +219,17 @@ def check_errors(self):
             return errors
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
+
+    def check_get_errors(self):
+        """Check for errors after having gotten a property.
+
+        Called if :code:`check_get_errors=True` is set for that property.
+        """
+        self.check_errors()
```

Comment:
Your remark in #667 made me realise that I didn't check the return value part of the interface. I think `check_get_errors` and `check_set_errors` should return what they get from `check_errors` (list of error entries as per the docstring). Otherwise that return just gets dropped on the way to the `check_get|set_errors` caller. One might argue that the only known callers are the properties, which don't make use of that return anyway, but for symmetry/least-surprise reasons it seems neater to me to return the values (and also codify that in the interface/abstract definitions).

Reply:
I answered in the general discussion, lest the discussion disappears in a resolved conversation.

Reply:
I think this is resolved now?

Reply:
Yes